staging/src/k8s.io/client-go/dynamic/client.go

```diff
@@ -205,12 +226,34 @@ func (rc *ResourceClient) DeleteCollection(deleteOptions *metav1.DeleteOptions,
 // Create creates the provided resource.
 func (rc *ResourceClient) Create(obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
 	result := new(unstructured.Unstructured)
-	err := rc.cl.Post().
-		NamespaceIfScoped(rc.ns, rc.resource.Namespaced).
-		Resource(rc.resource.Name).
-		Body(obj).
-		Do().
-		Into(result)
+	var err error
+
+	if !strings.Contains(rc.resource.Name, "/") {
+		err = rc.cl.Post().
+			NamespaceIfScoped(rc.ns, rc.resource.Namespaced).
+			Resource(rc.resource.Name).
+			Body(obj).
+			Do().
+			Into(result)
+	} else {
+		// If the provided resource is a subresource, the POST request should contain
+		// object name. Examples of subresources that support create verb:
+		//
+		//	core/v1/pods/binding
+		//	core/v1/pods/eviction
+		//	extensions/v1beta1/deployments/rollback
+		//	apps/v1beta1/deployments/rollback
+		resourceName, subresourceName := rc.parseResourceSubresourceName()
+		err = rc.cl.Post().
+			NamespaceIfScoped(rc.ns, rc.resource.Namespaced).
+			Resource(resourceName).
+			SubResource(subresourceName...).
+			Name(obj.GetName()).
```

Comment:
`obj` here is the subresource, but the name you want is the name of the parent resource, right?

Reply:
It seems subresource name is implicitly always the same as the resource name.